my-project/src/Form/ProductCategoryType.php

```diff
@@ -14,8 +14,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('name')
             ->add('description')
-            ->add('dateOfCreation')
```

Comment:
@mSobonVirtua to ma byc usuniete?

Reply:
jak najbardziej, coś chyba zepsułem znowu, bo nie mam tego u siebie na gałęzi

Reply:
jak najbardziej